### DISCUSSION TITLE
Small Window plexDIA Suggestions

### DISCUSSION THREAD

**Discussion by Aaron-Amunix** [vdemichev/DiaNN#449] [General]
I read the plexDIA paper and am excited about trying it with my experimental design.  I use small (4da) precursor mass windows for my current workflow and am getting great data using DIA-NN.  Will the plexDIA module of DIA-NN work with windows this small?  Will it hamper the ability to identify/quantify a peptide found in one or more of the channels if a labeled peptdiform is in a different precursor mass window?  Multiplexing DIA experiments seems like a gamechanger and I'm looking forward to trying it out!

-Aaron

**Comment by vdemichev** [comment]
Hi Aaron,

Any window size should work.

Best,
Vadim

**Reply by Aaron-Amunix** [reply]
Thanks Vadim, very excited to try this new module of DIA-NN.  I'm hopeful I'll decrease the already low CV with my experiments while increasing throughput.  Appreciate everything!
